client/jetpack-connect/remote-credentials.js

```diff
@@ -102,49 +106,69 @@ export class OrgCredentialsForm extends Component {
 	}
 
 	formFields() {
+		const { translate } = this.props;
+		const { isSubmitting, password, username } = this.state;
+
 		return (
 			<div>
-				<FormLabel htmlFor="username">{ this.props.translate( 'Username' ) }</FormLabel>
+				<FormLabel htmlFor="username">{ translate( 'Username' ) }</FormLabel>
 				<div className="jetpack-connect__site-address-container">
 					<Gridicon size={ 24 } icon="user" />
 					<FormTextInput
 						autoCapitalize="off"
 						autoCorrect="off"
 						className="jetpack-connect__credentials-form-input"
-						disabled={ this.state.submitting }
+						disabled={ isSubmitting }
 						id="username"
 						name="username"
 						onChange={ this.getChangeHandler( 'username' ) }
-						value={ this.state.username || '' }
+						value={ username || '' }
 					/>
 				</div>
 
 				<div className="jetpack-connect__password-container">
-					<FormLabel htmlFor="password">{ this.props.translate( 'Password' ) }</FormLabel>
+					<FormLabel htmlFor="password">{ translate( 'Password' ) }</FormLabel>
 					<div className="jetpack-connect__password-form">
 						<Gridicon size={ 24 } icon="lock" />
 						<FormPasswordInput
 							className="jetpack-connect__password-form-input"
-							disabled={ this.state.submitting }
+							disabled={ isSubmitting }
 							id="password"
 							name="password"
 							onChange={ this.getChangeHandler( 'password' ) }
-							value={ this.state.password || '' }
+							value={ password || '' }
 						/>
 					</div>
+					{ isSubmitting ? <Spinner /> : null }
 				</div>
 			</div>
 		);
 	}
 
+	renderButtonLabel() {
+		const { isFetching, isResponseCompleted, translate } = this.props;
+		const { isSubmitting } = this.state;
+
+		if ( isResponseCompleted ) {
+			return translate( 'Jetpack installed' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Already have Jetpack installed?' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).